packages/subgraph/schema.graphql

```diff
@@ -207,10 +239,9 @@ enum VotingMode {
 
 # TokenVoting
 
-type TokenVotingPlugin implements IPlugin @entity {
+type TokenVotingPlugin implements IPluginInstallation @entity {
   "TODO: attributes should be appended to Plugins once plugin is seperated from core"
   id: ID!
```

Comment:
if i understand correctly this id will be `psp installationId`, 
perhaps we need to add `pluginAddress` attribute to the interface or here.

if you agree consider this for the rest of the plugins.

Reply:
added `pluginAddress` to `PluginInstallation` and `pluginAddress` and `dao` to all the plugin specific entities